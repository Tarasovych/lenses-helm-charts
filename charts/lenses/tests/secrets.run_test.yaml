suite: test default values in secrets
templates:
  - secrets.yaml
tests:
  - it: should work
    asserts:
      # Check the rendered manifest has not changed
      # Review the changes and if they are verified use `helm unittest -u .` to update the manifest
      # Writing tests that cover this chaneg is highly recommended
      - matchSnapshot: {}
      - isKind:
          of: Secret
