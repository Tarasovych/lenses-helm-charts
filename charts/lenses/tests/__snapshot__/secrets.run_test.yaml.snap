should work:
  1: |
    apiVersion: v1
    data:
      client.keystore.jks: ""
      client.truststore.jks: ""
      keytab: ""
      lenses.keytab: ""
      lenses.opts.keystore.jks: ""
      lenses.opts.truststore.jks: ""
      lenses.tls.keystore.jks: ""
      lenses.tls.truststore.jks: ""
      saml.idp.xml: ""
      saml.keystore.jks: ""
    kind: Secret
    metadata:
      labels:
        app: RELEASE-NAME
        chart: lenses-4.1.7
        heritage: Helm
        lenses.io/app: RELEASE-NAME
        lenses.io/app.type: lenses-secret
        release: RELEASE-NAME
      name: RELEASE-NAME
    stringData:
      client.key.password: ""
      client.keystore.password: ""
      client.truststore.password: ""
      connect.clusters: ""
      jaas.conf: ""
      jvm.truststore.jks: ""
      jvm.truststore.password: ""
      lenses.opts.keystore.password: ""
      lenses.opts.truststore.password: ""
      lenses.tls.key.password: ""
      lenses.tls.keystore.password: ""
      lenses.tls.truststore.password: ""
      license: ""
      schema.registry.key.password: ""
      schema.registry.keystore.jks: ""
      schema.registry.keystore.password: ""
      schema.registry.password: ""
      schema.registry.truststore.jks: ""
      schema.registry.truststore.password: ""
      security.append.conf: ""
      security.conf: ""
